covasim/webapp/index.html

```diff
@@ -125,18 +127,18 @@
                 </div>
               </div>
               <div class="form-row mb-3">
-                <label for="datafile" class="col-12">Upload data</label>
-                <div class="col">
-                  <b-form-file id="datafile" name="datafile" accept=".csv, .xlsx" placeholder="<none>" v-model="datafile.local_path" @change="upload_datafile" :disabled="running"></b-form-file>
-                </div>
-                <div class="col-auto">
-                  <button id="reset-datafile" :disabled="running" class="btn btn-primary" type="button" @click="reset_datafile">
-                    Clear
-                  </button>
-                  <b-tooltip target="reset-datafile">
-                    Remove the input data file from the simulation
-                  </b-tooltip>
-                </div>
+                <!--                <label for="datafile" class="col-12">Upload data</label>-->
+                <!--                <div class="col">-->
+                <!--                  <b-form-file id="datafile" name="datafile" accept=".csv, .xlsx" placeholder="<none>" v-model="datafile.local_path" @change="upload_datafile" :disabled="running"></b-form-file>-->
+                <!--                </div>-->
+                <!--                <div class="col-auto">-->
+                <!--                  <button id="reset-datafile" :disabled="running" class="btn btn-primary" type="button" @click="reset_datafile">-->
+                <!--                    Clear-->
+                <!--                  </button>-->
+                <!--                  <b-tooltip target="reset-datafile">-->
+                <!--                    Remove the input data file from the simulation-->
+                <!--                  </b-tooltip>-->
+                <!--                </div>-->
```

Comment:
Will re-enable once automatic calibration is in place

Reply:
Instead of commenting it out, it would be better to have it behind a feature flag.

Reply:
Ah that's a good idea!